src/site/content/en/fast/signed-exchanges/index.md

```diff
@@ -325,6 +305,46 @@ Any site can cache, serve, and prefetch SXGs of the pages that it links to, wher
 ```
 [This article](/how-to-distribute-signed-http-exchanges/) illustrates how to use nginx to distribute SXGs.
 
+## Unique Advantages
+
+SXG is one of many possible technologies to enable [cross-origin prefetching](https://developer.chrome.com/blog/cross-origin-prefetch/). When deciding which technology to use, you may need to trade off between optimizing different aspects. The following sections illustrate a few of the unique values that SXG provides in the space of possible solutions.
+
+### Fewer requests to serve
+
+With cross-site prefetching, your server may need to serve additional requests. This corresponds to cases where a page was prefetched, but either the user didn't visit the page, or the prefetched bytes couldn't be shown to the user. For SXG, this additional traffic can be significantly reduced:
+
+- SXGs are cached and may be sent to users until they expire.
+- SXGs can be shown to users with cookies on your site.
+
+### Page speed improvement
+
+You may see additional page speed improvement from the following factors:
+
+- SXGs can be shown to users with cookies for your site.
+- SXG can also prefetch subresources for your pages, such as JavaScript, CSS, fonts, and images.
```

Comment:
'can also' or 'does also'?

Reply:
Fixed. Does, when specified using a `Link` header. (Technically, a link tag is sufficient when using sxg-rs or webpkg...)